helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -409,6 +409,7 @@ receivers:
         combine_field: attributes.log
         is_first_entry: '(attributes.log) matches {{ .firstEntryRegex | quote }}'
         max_log_size: {{ $.Values.logsCollection.containers.maxRecombineLogSize }}
+        combine_with: {{ .combineWith | default "\n" | quote }}
```

Comment:
Do we want to `quote` the value? A values.yaml entry like `combineWith: \t`, will provide the output `combine_with: "\\t"`. Maybe we should clarify a bit if the user needs to set literal or double-quoted scalar in values yaml if we quote in the template.

Reply:
I need an example as test. I wasn't sure.

Reply:
turns out it was a bit more involved than I thought!